util/saconfig

```diff
@@ -141,13 +163,14 @@ def main(args):
     logging.getLogger().setLevel(loglev)
 
     locator = options.locator
-    channels = set(options.calibrate + options.autozero + options.setzero)
-    axes = { str(c): {"range": [-1, 1], "unit": "m", "channel": c} for c in channels }
+    channels = set(options.calibrate + options.autozero + options.reference + options.setzero)
+    axes = {str(c): {"range": [-1, 1], "unit": "m", "channel": c} for c in channels}
 
     dev = smaract.MCS2(locator=locator, axes=axes, **CONFIG_3DOF)
-    
+
     calibrate(dev, options.calibrate)
-    autozero(dev, options.autozero)
+    reference(dev, options.autozero, mode=AUTO_ZERO_REF)
+    reference(dev, options.reference, mode=NORMAL_REF)
     set_zero(dev, options.setzero)
```

Comment:
Please, protect the calls to each of these function with a `if options.xxx:`.
Otherwise, we end-up calling each function. The functions eventually should do nothing, as no axes is passed, but it'd feel "safer".